rules/windows/defense_evasion_installutil_beacon.toml

```diff
@@ -1,8 +1,8 @@
 [metadata]
 creation_date = "2020/09/02"
-integration = ["endpoint", "windows"]
+integration = ["endpoint", "windows", "sentinel_one_cloud_funnel"]
 maturity = "production"
-updated_date = "2025/03/20"
```

Comment:
@w0rk3r this rule is a duplicate of  https://github.com/elastic/detection-rules/blob/fbfc696a86d3ec826fbd13fb8957d294f155dc17/rules/windows/defense_evasion_network_connection_from_windows_binary.toml#L131 (with more binaries) and should be deprecated.

Reply:
Added to my todo to review it, will handle it outside this PR